test/utils/TestUtil.scala

```diff
@@ -46,7 +46,7 @@ trait TestUtil extends UnitSpec with GuiceOneAppPerSuite with MaterializerSuppor
   }
 
   lazy val injector: Injector = app.injector
-  implicit lazy val mcc: MessagesControllerComponents = stubMessagesControllerComponents()
+  implicit lazy val mcc: MessagesControllerComponents = injector.instanceOf[MessagesControllerComponents]
```

Comment:
Now that this has been changed, can you remove the `bootstrap-play-26` dependency from the test scope?